src/app/wallets/get-balance-for-wallet.ts

```diff
@@ -0,0 +1,31 @@
+import { toLiabilitiesAccountId } from "@domain/ledger"
+import { LedgerService } from "@services/ledger"
+import * as Wallets from "@app/wallets"
+
+export const getBalanceForWallet = async ({
+  walletId,
+  lock,
+  logger,
+}: {
+  walletId: WalletId
+  lock?: DistributedLock
+  logger: Logger
+}): Promise<Satoshis | ApplicationError> => {
+  Wallets.updatePendingInvoices({
```

Comment:
We were wondering about this as well with 2 things:
- lock object locks on two different things (paymentHash vs walletId) for the two calls here, so it might already be inconsistent
- there are places where `getBalanceForWallet` is called without a `lock` arg entirely, so no value is passed forward to the two function calls

We decided to keep the implementation the same as the existing implementation for now, and potentially fix the inconsistencies later on (cc @bodymindarts)